==857503== Invalid free() / delete / delete[] / realloc()
==857503==    at 0x4839EAB: operator delete(void*) (vg_replace_malloc.c:584)
==857503==    by 0x1685D4: Arvore<int>::destroi() (arvore.cpp:77)
==857503==    by 0x142109: C_A_T_C_H_T_E_S_T_0() (arvore.cpp:86)
==857503==    by 0x130863: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14330)
==857503==    by 0x12FBCA: Catch::TestCase::invoke() const (catch.hpp:14169)
==857503==    by 0x12A92A: Catch::RunContext::invokeActiveTestCase() (catch.hpp:13025)
==857503==    by 0x12A6AE: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12998)
==857503==    by 0x129345: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12759)
==857503==    by 0x12C050: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13352)
==857503==    by 0x12D2C1: Catch::Session::runInternal() (catch.hpp:13562)
==857503==    by 0x12CFF4: Catch::Session::run() (catch.hpp:13518)
==857503==    by 0x16842C: int Catch::Session::run<char>(int, char const* const*) (catch.hpp:13236)
==857503==  Address 0x1ffeffeab0 is on thread 1's stack
==857503==  in frame #2, created by C_A_T_C_H_T_E_S_T_0() (arvore.cpp:81)
==857503== 
==857503== Invalid free() / delete / delete[] / realloc()
==857503==    at 0x4839EAB: operator delete(void*) (vg_replace_malloc.c:584)
==857503==    by 0x1685D4: Arvore<int>::destroi() (arvore.cpp:77)
==857503==    by 0x14232F: C_A_T_C_H_T_E_S_T_2() (arvore.cpp:94)
==857503==    by 0x130863: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14330)
==857503==    by 0x12FBCA: Catch::TestCase::invoke() const (catch.hpp:14169)
==857503==    by 0x12A92A: Catch::RunContext::invokeActiveTestCase() (catch.hpp:13025)
==857503==    by 0x12A6AE: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12998)
==857503==    by 0x129345: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12759)
==857503==    by 0x12C050: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13352)
==857503==    by 0x12D2C1: Catch::Session::runInternal() (catch.hpp:13562)
==857503==    by 0x12CFF4: Catch::Session::run() (catch.hpp:13518)
==857503==    by 0x16842C: int Catch::Session::run<char>(int, char const* const*) (catch.hpp:13236)
==857503==  Address 0x1ffeffeaa0 is on thread 1's stack
==857503==  in frame #2, created by C_A_T_C_H_T_E_S_T_2() (arvore.cpp:89)
==857503== 
==857503== Invalid read of size 4
==857503==    at 0x178D57: Arvore<int>::cria_arvore(int) (arvore.cpp:23)
==857503==    by 0x178D9F: Arvore<int>::cria_arvore(int) (arvore.cpp:31)
==857503==    by 0x1684DD: Arvore<int>::inicia(std::__cxx11::list<int, std::allocator<int> >&) (arvore.cpp:46)
==857503==    by 0x1684FC: Arvore<int>::inicia(std::__cxx11::list<int, std::allocator<int> >&) (arvore.cpp:48)
==857503==    by 0x142528: C_A_T_C_H_T_E_S_T_4() (arvore.cpp:102)
==857503==    by 0x130863: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14330)
==857503==    by 0x12FBCA: Catch::TestCase::invoke() const (catch.hpp:14169)
==857503==    by 0x12A92A: Catch::RunContext::invokeActiveTestCase() (catch.hpp:13025)
==857503==    by 0x12A6AE: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12998)
==857503==    by 0x129345: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12759)
==857503==    by 0x12C050: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13352)
==857503==    by 0x12D2C1: Catch::Session::runInternal() (catch.hpp:13562)
==857503==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==857503== 
